### DISCUSSION TITLE
cannot select newly created branch - it's grayed out

### DISCUSSION THREAD

**Discussion by davama** [netboxlabs/netbox-branching#108] [Q&A]
Hello,

trying out `v4.1.0` via docker.

Followed instructions on the [README](https://github.com/netboxlabs/netbox-branching?tab=readme-ov-file#netbox-branching) but cant seem to be able to select a new branch.

After creating a single site, I created the branch as shown in the picture, but I am not able to select it and work off of it.

![2024-09-10_15-27](https://github.com/user-attachments/assets/21407edc-f74a-40d1-9f36-8b1942afbd5c)

What could I be missing?

Thanks!

**Comment by cruse1977** [comment]
Creating a branch creates a job, what does the job and status of the branch
say ?

**Comment by davama** [comment]
Ah
my worker was not up
That was the issue
Thank you!
